p2p/discover/v5wire/msg.go

```diff
@@ -61,7 +61,7 @@ type (
 		Nonce Nonce
 	}
 
-	// WHOAREYOU contains the handshake challenge.
+	// Whoareyou contains the handshake challenge.
```

Comment:
I think this is deliberate, and @fjl wants it like this. `WHOAREYOU` is a protocol message. How come you only changed the `WHOAREYOU`, but not `PING` on L78 and `PING` on L84 etc? They should all use the same format

Reply:
>  But it's not super important to me to keep it that way especially if it is inconsistent.

Well, as of yet it's kind of consistent, but this PR makes it inconsistent (since PING/PONG) is untouched